services/user/Homepage/ui/src/components/form-profile.tsx

```diff
@@ -0,0 +1,116 @@
+import { useForm } from "react-hook-form";
+import { z } from "zod";
+import { zodResolver } from "@hookform/resolvers/zod";
+import { Button } from "@/components/ui/button";
+import { Input } from "@/components/ui/input";
+import {
+    Form,
+    FormControl,
+    FormDescription,
+    FormField,
+    FormItem,
+    FormLabel,
+    FormMessage,
+} from "@/components/ui/form";
+
+const formSchema = z.object({
+    displayName: z
+        .string()
+        .min(2, "Display name must be at least 2 characters"),
+    bio: z.string().max(40, "Bio must be less than 40 characters"),
```

Comment:
Feels like this should be longer, no?

Reply:
Just copied X's, they've got 50 for the name and 160 for the bio.